scripts/ci/retry.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
Instead of having a custom script to retry, what about using a [GH action retry step](https://github.com/marketplace/actions/retry-step)?